app/lib/package/backend.dart

```diff
@@ -757,6 +757,7 @@ class PackageBackend {
       replacedBy: pkg.replacedBy,
       latest: latest.toApiVersionInfo(),
       versions: packageVersions.map((pv) => pv.toApiVersionInfo()).toList(),
+      advisoriesUpdated: pkg.latestAdvisory,
```

Comment:
nit: should we make `advisoriesUpdated` `final`?

Reply:
done